exonum/src/api/node/mod.rs

```diff
@@ -247,8 +247,8 @@ impl SharedNodeState {
 
     pub(crate) fn shutdown_broadcast_server(&self) {
         let state = self.node.read().expect("Expected read lock");
-        for address in state.broadcast_server_addresses.iter() {
-            address.do_send(websocket::Terminate);
+        for server in state.broadcast_server_addresses.iter() {
+            server.do_send(websocket::Terminate);
```

Comment:
Maybe `connection` ? What do you thinks? IMO, connection is more appropriate name here.

Reply:
I don't have a preference for the naming. I follow the original name, server, before the refinement. However, I also can change it to connection. These two names, server or connection, are fine to me both.